build.sbt

```diff
@@ -98,14 +99,18 @@ lazy val tck = project
 lazy val codegenCore =
   project
     .in(file("codegen/core"))
-    .enablePlugins(PublishSonatype)
-    .dependsOn(sdk)
+    .enablePlugins(AkkaGrpcPlugin, PublishSonatype)
     .settings(
       name := "akkaserverless-codegen-core",
       testFrameworks += new TestFramework("munit.Framework"),
       Test / fork := false
     )
     .settings(Dependencies.codegenCore)
+    .settings(
+      Compile / akkaGrpcGeneratedLanguages := Seq(AkkaGrpc.Java)
+    )
+    .settings(crossScalaVersions := Dependencies.ScalaVersionForCodegen,
+              scalaVersion := Dependencies.ScalaVersionForCodegen.head)
```

Comment:
it's a bit unfortunate that we need to force these modules to use Scala 2.12 so they can be used 'reflectively', but OK I suppose ;).